### PR TITLE
Issue/21 server database

### PR DESCRIPTION
**Pull request by AnnaGsell**

### CODE REVIEW

File: server/cmd/models.go [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+)
+
+// Study holds the configuration data and is refered by multiple users.
+type Study struct {
+	gorm.Model
+	ID                                int
+	ServerURL                         string
+	StudyName                         string
+	NotificationText                  string
+	IsSpartalTaskEnabled              bool
```
Comment (mac641) [inline server/cmd/models.go:18]:
```suggestion
	IsSpatialSpanTaskEnabled              bool
```

File: server/cmd/models.go [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+	IsPsychomotorVigilanceTaskEnabled bool
+	IsReactionGameEnabled             bool
```
Comment (mac641) [inline server/cmd/models.go:20]:
```suggestion
	IsMentalArithmeticEnabled             bool
```

File: server/cmd/models.go [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+// UserLog binds the data from an execution flow to an user and a timestamp.
+type UserLog struct {
+	gorm.Model
+	UserID              int
+	MoodID              int
+	ActivityID          int
+	AccessMethodID      int
+	SSTResultID         int
+	PVTResultID         int
+	ReationGameResultID int
```
Comment (mac641) [inline server/cmd/models.go:79]:
```suggestion
	MentalArithmeticResultID int
```

File: server/cmd/models.go [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+
+// PVTResult holds the pvtResult value.
+type PVTResult struct {
+	gorm.Model
+	ID             int
+	PVTResultValue int
+}
+
+// ReationGameResult holds the Result of Reaction time measuring game
+type ReationGameResult struct {
```
Comment (mac641) [inline server/cmd/models.go:142]:
```suggestion
type MentalArithmeticResult struct {
```

**Review by diehmlukas — DISMISSED** [review]

**Review by mac641 — CHANGES_REQUESTED** [review]
Detailed change request in comments ^^

**Review by mac641 — APPROVED** [review]
👍🏿

**Review by diehmlukas — APPROVED** [review]

**Pull request merged**